disco_aws_automation/disco_elasticsearch.py

```diff
@@ -416,3 +420,7 @@ def get_hostclass_option(self, option, hostclass):
     def get_hostclass_option_default(self, option, hostclass, default=None):
         """Fetch a hostclass configuration option, if it does not exist get the default"""
         return self.get_aws_option_default(option, hostclass, default)
+
+    def _get_nat_eips(self):
+        env_option = 'envtype:{}'.format(self.environment_name)
+        return self.config_vpc.get(env_option, 'tunnel_nat_gateways')
```

Comment:
Good question. 

If option *tunnel_nat_gateways* is there but empty, things will work (empty string will be appended to the list of whitelisted IPs).

If *tunnel_nat_gateways* is not in disco_vpc.ini, it will fail with **ConfigParser.NoOptionError: No option 'tunnel_nat_gateways' in section: 'envtype:ci'**